content/License.txt

```diff
@@ -1,58 +0,0 @@
-
```

Comment:
Yeah that's a no-no. Can we just keep 2 repos but still change away from sake to MSBuild? I get that it isn't pretty, but this stuff here almost never changes anyway.